Dockerfile

```diff
@@ -33,10 +33,8 @@ RUN conda install --name python2 --quiet --yes -c bioconda zeroc-ice && \
         gseapy \
         py2cytoscape \
         pydot \
-        tqdm
-
-# Add idr-notebook library to path
-RUN echo /notebooks/library > /opt/conda/envs/python2/lib/python2.7/site-packages/idr-notebooks.pth
+        tqdm \
+        git+https://github.com/IDR/idr-py@develop
```

Comment:
Travis failed, this needs to be `@master`

Reply:
I assumed it would be develop, cause am opening the PR against develop

Reply:
ahh wrong repo branch, yeah will change it to master